nestkernel/event_delivery_manager.cpp

```diff
@@ -446,7 +442,6 @@ EventDeliveryManager::gather_spike_data_( const thread tid,
         decrease_buffer_size_spike_data_ = false;
       }
     }
-#pragma omp barrier
 
```

Comment:
I agree that we don't need this barrier since gather_completed_checker_.any_false() also invokes a barrier.